.dependabot/config.yml

```diff
@@ -0,0 +1,13 @@
+# https://dependabot.com/docs/config-file/
+version: 1
+update_configs:
+  # Keep package.json (& lockfiles) up to date as soon as
+  # new versions are published to the npm registry
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "live"
+    # Apply default reviewer and label to created pull requests
+    default_reviewers:
+      - "XhmikosR"
```

Comment:
@mozfreddyb you want me to add you here?

Reply:
yup

Reply:
no need, if you think you can take it on your own

Reply:
I don't watch the repo, so I assume this will notify me. Just be careful with stuff that might affect the server since we don't test this on CI.